x-pack/legacy/plugins/infra/public/containers/logs/log_analysis/log_analysis_setup_state.tsx

```diff
@@ -60,13 +61,30 @@ export const useAnalysisSetupState = ({
     return cleanupAndSetupModule(selectedIndexNames, startTime, endTime);
   }, [cleanupAndSetupModule, selectedIndexNames, startTime, endTime]);
 
-  const validationErrors: ValidationErrors[] = useMemo(
-    () =>
-      Object.values(selectedIndices).some(isSelected => isSelected)
-        ? []
-        : ['TOO_FEW_SELECTED_INDICES' as const],
-    [selectedIndices]
-  );
+  const validationErrors = useMemo<string[]>(() => {
+    if (selectedIndexNames.length === 0) {
+      return [
+        i18n.translate(
```

Comment:
Can we try to keep the translations in the presentational components? This is why the errors are encoded using IDs on the API in the first place.